content/ch-applications/facial-expresson-recognition.ipynb

```diff
@@ -0,0 +1,808 @@
U7yJik8FsuTC2RgvCYYjYroiyhTCkk2snNP/MFmidn79KhmJQYX1xyfPqEyw+/RgZiHzAJJVekpO/MaoCaGEGckKqGH/vuf4Fv/WPfz+jsTDuNGNmenPATv+2fpXYTUgpqqeMcRhIXH3qNzckx49OzdxWw7c2bXH3iGxFTdrEmYSqrwXDesj4+5q//7t/J4ZffpH74mNXxIY/u3SFmvT/ULy/qex/XyHhCQdawkjl5800m70in1fefmZ5fcvftxzx5+Za+/xJbYb3DOnUryCGV2QOkELRLTupqkpNq/kMMYCHaneeEMtTC0F8XoBQIsSMWSUGMWeUDGHLImJg1xTUNGqkAygosHocJdb0IEZ0tOcNgIBhLiCWSwliWfUsbtDjuhviS9d7MuwhTo0xNa0Tjw3PGOltmlYbGj3HJIkMAHCHDkDNDjFhXkVCmb9tHQt8p81GgajS80BiH9yPdbNmaZjSBDP0wkEQIYmjDwGRc8/jilGlh6GXjaaY1l5cPiYNC3q6p6IeOEIa9B91uTRRgp3E0uxmRfrPMeq5pYGWqXv6l0LpW2Wu90a7oXuuWrn9eHeLLn19vW8T7sBhlNKyuNI7X31AjOtp1uze+yJsti4sr3n7zPqC4dkqxVHehqmrqqmLoOz7xLR9n27VcnJ9TNzWYuuS7C5V1WO8JCU4fnzFuGs7Oz3jhxTv4owmSKr785Ue8+NJtDuaZ1PXU8zmSwFqHqWrtAGIktFucsSw3G/qhQ+oKK57GWNUiOPVyixl6EoNkFqslo9phvcV5h68qYsxsV0uMzVxdXGCzIVs1XMUUm/luw5OLc4yriETatiX1SXeUIfJCNzAA1Xt8ztEYDi8WPIrQpoy14EvrblCqaGp7Ug3OWoRMSoMW0OJOZUCjk0WthSTrXCmnpBHQkkuUgkGC7nzFGkhaEFIIulDkDOIQW0xNjN0brEq5JtT5QAPenrkc1PAx7eAFKWa5+jyTx0/JzkJhvT13nTnLvO+5mo5g2ykkWaHi1CEgGJ19iGC8J+SEsZb2+CZ/+1/6fRy//kXGT57S3rjJ4oMf1EG8STrMTzuwXtuEH/0d3823ff8fY3R6SirQzebmTX70d/8uxDtMKl2CV2jLJJQlZyLj+QGresTZvbuEGDAxUtUjIoCxxJL9ZRqPq7wyuSSCRGanKgx+L4gwWcv0/JLHH7ijUJMRvLH4Mh/Sz9KoS1HKex/ILOqeoOwyjRcxCb0+pQzTU9brJAZ1v8iBbA2h25K6rkRyV8SoZAUlK1mMccQcr3fuRv0PU8oMKapTh6BQXdwZwQZNRMVwdb4kdAPWWvoQkKzFNKMEi0QGK1gxOOvKNarwpukTI+8Z1ZUyQ506u0gGGSIki1g1OI7J0G03SNQlPBuQypOdRVyNq6a0IakMwjtSVosuM67pUiInhUsnriKEyLpr8c6Re2WuqrsE9H1Qwe/OOmn3uZTuk7Rz1NaiCxQ9Vpnx7OvG89CazmSv7xnNKLq2BtPXumbRFR6lIiIplZyqr/143xUjUHovZSawwzyvLVWusdGcNcI494FnP0mt6sI29nStKNNsdshP/MSPc/70dK8uBp1/OOeoqxp7MCPFnm5I1LWyiZ48PsVIZjyZsFlv2Gwn2BQZN5VCOs7hKoe6LRerECBve0IfGNdTxFWQIlUWJEGXIhahykJtDMNmTZhNqPwYaxUyEOs00rz3PH18Stu3VJVDUCr3dHLIaD7nSdfRt1tlBuGgKgyfmPmSAfcuwLi875g4P5wpG9Gop9aQMjZoAqczQErESMGMswpZy4KQCrQToqgTOGoXEnf+WVa7IzC6M4+BkDtEPFkyZijhfMZB5dXa36jaW0XOO8NQhchMlmIjrXOrnRbKZIM1GjIXcyiDYZAs9PfuIO9RiEAhsu7FWzij85Y89uRcIUXLYa1GPsgAOUaGoaeuaw6+8HlGZ6dsb93k/s/9dqrxRBntIkVv1mG8zsMMAzHD+uSEH/y+7+Pwi19kcnrK6uSIxUc/ipiScouBEiVCUtaarRx50zP59Oc5fvyIq9mUs5dfwk2caoGyLvg4y9D1GJuxLkLWRTgD7c3jr+jxZ2JkeeNIIw/KZ2bKIpNS0cyIQJkpAQXGsgzDQBr0ebWryHsHhlRmgzZpUVJEz5YO2JLEqYedMRpD79SJArFKRVdLESUZ9GoarCLqjBFLINFHnbfuiF3iHSlmhr5l1a6IJGJhfqbd4lw2qLYw+EKKZKuQ5o7v3DSG6axmCD2+FsRahjaQjQXvCSHRVCPW2035PJRQOpnPMaOxetSJZZNha3RtqJ0jC3R9Vr8+I9Qltn6FsFgsqQrN/emDp3SrlVp7iYEh7DcGz8JyO++53ZqXC1JQvNEVKt8RLriG8HZQnHZV1z1Tzu/omnavQSlCZRZMzkXe8fUd78tiBNfFJz3373LsPpiCge4gvR02mop1EKXFPzo64vUvfInzJ6dKMy7PtMtIct7hnEEcgGG97bl1csJ6vUWyMJ7UVE3DxdWa0XTGjeMp2TcMGbZdr3i8MQxxIJFoNx05OWajI3SCoaJWQ6Lve6q8E/ZaKlsxENnmzKj47UmhkI7mI55eXJKLRf90MiGT6UPk/OyKk2y5eXzMxdUFy+1ACnpR+boCE/mhHHnTCK+UmdHuCAKn8ylvvHiPxqKLsVFqsYlaoK1Rby6sLgCkiDcGiaEovhPiHNkIEiLWWoXtrCNKIoaIIDhAclKHBQSJFmdBkjLjoiTIUdX+We+QlJMKbTOqu0I7OYmZZFLRzCiev4fRJGMLay8lPcfqcXeD6vHT5z3ujKG9fYPtxz9EMwR63TrrNVY7TDUnRUFSoaOHwPhywc///j/F+OLiGXjumB//nb+T4e5dXRxLAFvMWgwVWlHo7+Dzn2d8dsr61i0WH3wVby1khbGMdeA8kiNCRSeZyWrJN/yh/5Dx2fn+9TYnx/zId/9zbI8O0aTUAStJf85ptLbBEqMQc+L81XtsTg6ZnJ4/B9UlI6
```

Comment:
Hey I think this latex code is not rendering properly.

Reply:
@divshacker Thank you for your comment! I'm checking it right now.

Reply:
@divshacker This is strange because with Jupyter-Lab (2.3.1 on Windows 10, with Python 3.8.5) I can visualize everything correctly. I have re-wrote the formulas in one line (previously most formulas were spread on multiple lines). I still can visualize everything correctly. Can you tell me if it's ok now? Thank you very much

Reply:
Yes it is ok now